### PR TITLE
Use `StringNumber` for integers beyond `i32` - part3

### PR DESCRIPTION
**Pull request by marioCluml**
Closes: #262 ;

### CODE REVIEW

File: src/graphql/account.rs
Comment (henry0715-dev) [inline src/graphql/account.rs:559]: There is no need to change the part used for paging.

Comment (marioCluml) [reply src/graphql/account.rs]: Done, removed `StringNumber` from paging for `clusterTotalCount` and `accountTotalCount`.

File: src/graphql/cluster.rs
Comment (henry0715-dev) [inline src/graphql/cluster.rs:297]: `//` It would be better to change to `///`.
And it is recommended to modify the document to end with a "."

**Comment by henry0715-dev** [comment]
I think it would be better to modify the Title.
Use `StringNumber` for integers beyond `i32`
example - https://github.com/aicers/review-web/pull/265

File: src/graphql/event/network.rs
Comment (henry0715-dev) [inline src/graphql/event/network.rs:85]: "representable" appears to be a typo.

Comment (marioCluml) [reply src/graphql/event/network.rs]: Done, fixed the typos

**Review by henry0715-dev — APPROVED** [review]
👍

**Comment by sophie-cluml** [comment]
Could you rebase this please?

File: CHANGELOG.md
Comment (sehkone) [inline CHANGELOG.md:78]: @marioCluml @henry0715-dev 

The previously merged commits documented this in the section `Changed`, but this did in the section `Fixed`. If we consider this a bug we need to put all into `Fixed`, but I think it is still Ok to keep it in the `Changed`.  Either way, we have to make it consistent.

Because the correction of README can be done right before releasing a new version, I am merging this as it is. And, I will take care of the README later.

**Review by sehkone — APPROVED** [review]

File: CHANGELOG.md
Comment (marioCluml) [reply CHANGELOG.md:78]: Yes because we were merging it sequentially I was going to rebase this branch on top of main after part 2 was merged and in doing so fix the CHANGELOG that has part 2 CHANGELOG entry. Then for part 4 I was going to do the same process as well. So that way it will be more consistent while being merged sequentially.

Comment (marioCluml) [reply CHANGELOG.md:78]: I'll be back home in about an hour or so and can fix it then since part 2 was merged earlier today. I was waiting for the part 2 to be merged before fixing this

Comment (sehkone) [reply CHANGELOG.md:78]: @marioCluml 

I suggest fixing the README in #270 instead of this. By doing so, I think we will not need to enumerate all the items and would rather mention like "all the relevant APIs" if we cover all.

**Pull request merged**
